Bugzilla/TCT.pm

```diff
@@ -0,0 +1,100 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::TCT;
+use 5.10.1;
+use Moo;
+
+use Bugzilla::DaemonControl qw( on_finish on_exception );
+use File::Temp;
+use Future::Utils qw(call);
+use Future;
+use IO::Async::Process;
+
+use constant TCT_BIN => '/usr/local/bin/tct';
```

Comment:
this belongs in `localconfig`, with all the other paths to binaries/scripts.

Reply:
And here I realize that every path listed in localconfig might now actually work in the new environment, when using LOCALCONFIG_ENV=1